source/extensions/filters/network/redis_proxy/conn_pool_impl.cc

```diff
@@ -122,6 +142,85 @@ InstanceImpl::ThreadLocalPool::makeRequest(const std::string& key,
     client->redis_client_->addConnectionCallbacks(*client);
   }
 
+  // cache host for quick lookup via address for redirection support
+  if (host->address()) {
+    auto hostCachedByAddress = host_address_map_.find(host->address()->asString());
```

Comment:
`host_cached_by_address`. Check elsewhere for snake vs. camel case.

Reply:
ok ... will check all of my code for snake vs. camel case